lib/keycloak.rb

```diff
@@ -269,7 +305,7 @@ def user_sign_out(refresh_token)
 
     private
     def user_by_token(token)
-      @users.find{|_, user| user[:token] == token}
+      @users.find{|_, user| user[:token] == token}[1]
```

Comment:
I still don't get that code. @users is a hash of hashes, what you expect to get from `{username: email, email: email, ... }[1]` ?

Reply:
Oh stupid me - the result of find here is `[key,value]` :)